src/app/app-routing.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { HomeComponent } from './components/home/home.component';
+import { MainComponent } from './components/main/main.component';
+import { ProfileComponent } from './components/profile/profile.component';
+import { Page404Component } from './components/page404/page404.component';
+
+const routes: Routes = [
+  { path: '', component: HomeComponent },
+  { path: 'main', component: MainComponent},
+  { path: 'profile', component: ProfileComponent},
```

Comment:
Be careful... Remember avoid adding fractures that are out of the scope of the PR. Is it really needed the Profile component and router for login?